test/e2e/src/pom/videoPlayerExamplesPage.ts

```diff
@@ -0,0 +1,27 @@
+import { Page } from '@playwright/test';
+
+/**
+ * Video player examples page object
+ */
+export class VideoPlayerExamplesPage {
```

Comment:
Suggested to change this from `ExamplesPage` to `MainPage`, i.e. we have many example pages, but this refers to the main one specifically

Reply:
Renamed to mainPage